### DISCUSSION TITLE
Block selector icon bug

### DISCUSSION THREAD

**Discussion by AbbottMc** [Mojang/minecraft-editor#182] [Bugs]
The block icon of block selector and hotbar is unfit, it seems using the block icon image directly (?

![image](https://github.com/Mojang/minecraft-editor/assets/16969459/2e245c86-9f03-4f91-8dad-dc4f3c8fa4bc)

**Comment by m-brisch** [comment]
Thanks for the feedback @AbbottMc! We're aware of the issue and should have a fix around end of the month

**Reply by AbbottMc** [reply]
Hooray!
